Week-1/Code/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+
+        
+            <div class="sidebar">
+                <ul>
+                    <li class="fa fa-bars" />
+                    <li class="fa fa-home" />
+                    <li class="fa fa-search" />
+                    <li class="fa fa-volume-up" />
+                    <li class="fa fa-user" />
+                    <li class="fa fa-cog" />
+                    <li class="fa fa-spotify" />
+                    <li class="fa fa-soundcloud" />
+                </ul>
+            </div>
+
+
+            <div class="firstblock">
+
+                <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg">
+
+
+                <div>
+                    <div class="text">
+                    <h1>Cattyboard Top 100 single charts(11.06.36)</h1><br>
+                    <h6>unknown artist</h6>
+                    <h6>2016.charts.100 songs</h6>
+                    </div>
+                    <ul class="btn">
+                    <li class="fa fa-play">Play all</li>
+                    <li class="fa fa-plus">Add to</li>
+                    <li class="fa fa-ellipsis-h">More</li>
+                    </ul>
+                </div>
+
+            </div>
+
+
+            <div class="list">
+                <table>
+                    <tr>
+                        <td>1.One Dance</td>
+                        <td>Crake feat Catkid & Cyla</td>
+                        <td>2:54</td>
+                        <td>CATTY CLOUD SYNC</td>
+                    </tr>
+                    <tr>
+                        <td>2.Panda</td>
+                        <td>Cattee</td>
+                        <td>4:06</td>
+                        <td>CATTY CLOUD SYNC</td>
+                    </tr>
+                    <tr>
+                        <td>3.Can't stop the feeling</td>
+                        <td>Catin Cimberlake</td>
+                        <td>3:56</td>
+                        <td>CATTY CLOUD SYNC</td>
+                    </tr>
+                    <tr>
+                        <td>4.Work from Home</td>
+                        <td>Cat Harmont feat Colla</td>
+                        <td>3:34</td>
+                        <td>CATTY CLOUD SYNC</td>
+                    </tr>
+                </table>
+            </div>
+
+                <footer>
+                    <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg">
+                    <div>
+                        <h3>If it Ain't Love<br>
```

Comment:
please follow bem naming conventions for classes u can refer : http://getbem.com/naming/